x/auction/keeper/grpc_query.go

```diff
@@ -37,7 +37,7 @@ func (q Querier) RewardsAuction(goCtx context.Context, msg *auction.QueryRewards
 	bid, id := q.Keeper(&ctx).getRewardsBid(msg.Id)
 	r := &auction.QueryRewardsAuctionResponse{Id: id}
 	if bid != nil {
-		r.Bidder = sdk.AccAddress(bid.Bidder).String()
```

Comment:
this should work

Reply:
we already have bech32 string ,so if again we convert string -> sdk.AccAddr -> string (it will get different address)

Reply:
I'm saying  , if we keep `bid.Bidder` as `bytes` then `sdk.AccAddress(bid.Bidder).String()` should work correctly.